src/components/checkbox/README.md

```diff
@@ -0,0 +1,18 @@
+## Checkbox component
+
+The implementation requires controlled behaviour, so the component requires to be wrapped
+and provided with state and handler.
+
+### Props:
+
+```typescript
+interface CheckboxProps {
+  checked: boolean
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void
+  id?: string
+  label?: string
+  labelPosition?: "left" | "right"
+  ref?: React.MutableRefObject<HTMLInputElement>
```

Comment:
Also, should we expose the `ref` and `className`? I mean we can expose all the underling properties of course but maybe we should agree on which are useful for us and avoid the rest. We can agree on a common set of required props like for example:
`id` and `className`

Reply:
`ref` could be exposed so the outer code can manipulate it (with useRef etc). `className` is exposed so that the Checkbox will be wrappable in styled()

Reply:
Ok. So @builat and @Azein lets agree that all our components will have:
`id`, `ref` and `className`